src/titiler/extensions/titiler/extensions/render.py

```diff
@@ -0,0 +1,182 @@
+                doseq=True,
+            )
+
+        def _prepare_render_item(
+            render_id: str, render: Dict, request: Request, src_path: str
+        ) -> Dict:
+            """Prepare single render item."""
+            query_string = _prepare_query_string(render, src_path)
+            links = [
+                {
+                    "href": factory.url_for(
+                        request,
+                        "Show STAC render",
+                        render_id=render_id,
+                    )
+                    + "?"
+                    + urlencode({"url": src_path}),
+                    "rel": "self",
+                    "type": "application/json",
+                    "title": f"{render_id} render item",
+                },
+                {
+                    "href": factory.url_for(
+                        request,
+                        "tilejson",
+                        tileMatrixSetId="{tileMatrixSetId}",
+                    )
+                    + "?"
+                    + query_string,
+                    "rel": "tilesets-map",
+                    "title": f"tilejson file for {render_id}",
+                    "templated": True,
+                },
+                {
+                    "href": factory.url_for(
+                        request,
+                        "wmts",
+                        tileMatrixSetId="{tileMatrixSetId}",
+                    )
+                    + "?"
+                    + query_string,
+                    "rel": "tilesets-map",
+                    "title": f"WMTS service for {render_id}",
+                    "templated": True,
+                },
+            ]
+
+            return {
+                "params": render,
+                "links": links,
+            }
+
+        @factory.router.get(
+            "/renders",
+            response_model=RenderItemList,
+            response_model_exclude_none=True,
+            name="List STAC renders",
+        )
+        def render_list(request: Request, src_path=Depends(factory.path_dependency)):
+            with factory.reader(src_path) as src:
+                renders = src.item.properties.get("renders", {})
+            prepared_renders = {
+                render_id: _prepare_render_item(render_id, render, request, src_path)
+                for render_id, render in renders.items()
+            }
+            return {
+                "renders": prepared_renders,
+                "links": [
+                    {
+                        "href": str(request.url),
+                        "rel": "self",
+                        "type": "application/json",
+                        "title": "List Render Items",
+                    },
+                ],
+            }
+
+        @factory.router.get(
+            "/renders/{render_id}",
+            response_model=RenderItemWithLinks,
+            response_model_exclude_none=True,
+            name="Show STAC render",
```

Comment:
what do you think about `STAC Renders metadata` ?

Reply:
Renamed